frontend/nuxt.config.js

```diff
@@ -77,6 +78,22 @@ export default {
     icon: false,
     workbox: {
       // Configure your service worker options here
+      cacheAssets: false,
```

Comment:
I feel that all this config lines could be a bit risky from a perfomance and introduce maybe some not desired side-effect.
I propose to not include this part, at least initally, and keep the other changes.
What do you think?

Reply:
yea we don't have to include